src/functionalTest/java/uk/gov/hmcts/reform/juddata/cameltest/JrdBatchAuditingAndEmailTest.java

```diff
@@ -78,7 +78,7 @@ void testParentOrchestrationFailureEmail() throws Exception {
         //verify(emailService, times(1)).sendEmail(any(), any());
     }
 
-    @Test
+    //@Test
```

Comment:
Can these two Tests be removed instead of commented out? They do not seem to serve any purpose - but if we require them for some reason, would it be better to add `@Ignore("With a reason why")`?